src/hooks/useKeyLongPressed.ts

```diff
@@ -1,29 +1,29 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState } from "react";
 
 export const useKeyLongPressed = (): [string] => {
-  const [keyLongPressed, setKeyLongPressed] = useState('');
+  const [keyLongPressed, setKeyLongPressed] = useState("");
 
   useEffect(() => {
     let timerId: ReturnType<typeof setTimeout>;
 
     const handleKeyDown = (event: KeyboardEvent) => {
-        timerId = setTimeout(() => {
-          setKeyLongPressed(event.key);
-        }, 300);
+      timerId = setTimeout(() => {
+        setKeyLongPressed(event.key);
+      }, 300);
     };
     const handleKeyUp = () => {
       clearTimeout(timerId);
-      setKeyLongPressed('')
+      setKeyLongPressed("");
     };
 
-    document.addEventListener('keydown', handleKeyDown);
-    document.addEventListener('keyup', handleKeyUp);
+    document.addEventListener("keydown", handleKeyDown);
+    document.addEventListener("keyup", handleKeyUp);
     return () => {
       clearTimeout(timerId);
-      document.removeEventListener('keydown', handleKeyDown);
-      document.removeEventListener('keyup', handleKeyUp);
+      document.removeEventListener("keydown", handleKeyDown);
```

Comment:
why is this file changed?

Reply:
It is just changed because of lint, the only change is single quotes to double quotes